### ISSUE TITLE
Buttons have incorrect styles in 7.1.0

### ISSUE THREAD

**Issue by edenman** [firebase/FirebaseUI-Android#1874]
### Step 2: Describe your environment

  * FirebaseUI version: 7.1.0
  
#### Steps to reproduce:

  1. Customize button appearance by overriding styles
  2. Update from 6.2 to 7.1
  3. Buttons are no longer styled appropriately
 
Here's how I'm doing the customization:
  ```
  <style name="FirebaseUI.Button.AccountChooser.GoogleButton">
    <item name="drawableStartCompat">@drawable/ic_sign_in_google</item>
    <item name="android:background">@color/auth_button_bg_google</item>
    <item name="android:textColor">@color/actually_white</item>
  </style>
  ```
I checked and the style hasn't changed names or anything:
https://github.com/firebase/FirebaseUI-Android/blob/aef1d2f94901295a6f7b66eeea75f1ec7a90cd37/auth/src/main/res/values/styles.xml#L231

Looks like this is happening because of a switch from `com.firebase.ui.auth.util.ui.SupportVectorDrawablesButton` to `Button` in 5342e3dc69221069745945ef613be7eeaf0573eb .  I can get it working again if I use `drawableLeft` but obviously that's not ideal for RTL users.  What was the motivation for that change?  It seems to have even broken the default UI.  If I remove all my style customizations, here's what the screen looks like:
![Screenshot_1606165952](https://user-images.githubusercontent.com/1063557/100015965-973a5f80-2e3d-11eb-9d5e-1f39d493a4a5.png)

**Comment by DeweyReed** [comment]
Same here. It seems MDC changes `<Button>` to `MaterialButton`, and then styles are lost.

**Comment by samtstern** [comment]
@edenman @DeweyReed thanks for reporting this.  That change was just a cleanup, if it's breaking we can revert it for version `7.1.1`.

**Comment by samtstern** [comment]
This has been fixed and released in version `7.1.1`

**Comment by edenman** [comment]
Confirmed: fixed in 7.1.1
